test/e2e/const.go

```diff
@@ -83,7 +89,12 @@ const (
 	GiteaUserNameVar     = "GITEA_USER_NAME"
 	GiteaUserPasswordVar = "GITEA_USER_PWD"
 
+	RKE2VersionVar = "RKE2_VERSION"
+
 	CapaEncodedCredentialsVar = "CAPA_ENCODED_CREDS"
+	AzureSubIDVar             = "AZURE_SUBSCRIPTION_ID"
+	AzureClientIDVar          = "AZURE_CLIENT_ID"
+	AzureClientSecretVar      = "AZURE_CLIENT_SECRET"
```

Comment:
This 3 variables should be added to env in `e2e-long.yaml` so the post submit and nightly won’t fail after this gets merged.

Reply:
These have been added to the GHA secrets.

Reply:
Yes you are right, adding now.